claims.go

```diff
@@ -112,10 +115,25 @@ func validateBoundClaims(logger log.Logger, boundClaims, allClaims map[string]in
 
 	scan:
 		for _, v := range expVals {
-			for _, av := range actVals {
-				if av == v {
-					found = true
-					break scan
+			if useGlobs {
+				vs, ok := v.(string)
+				if !ok {
```

Comment:
We've already verified this at role update/create time, right?  I think `vs := v.(string)` is appropriate. We'd panic if not string, but that's OK since the role code shouldn't never let non-strings become expected values.

Reply:
Addressed in the third commit.